src/index.ts

```diff
@@ -1695,36 +1695,35 @@ async function getCrossChainOptionsForLink(
 	tokenType: number
 ): Promise<Array<interfaces.Chain & { tokens: interfaces.Token[] }>> {
 	if (tokenType > 1) {
-		console.log("Can't bridge link type")
-		return []
+		throw new Error('Can not bridge link type')
 	}
 
 	const supportedChains = await getSquidChains(isTestnet)
 
 	const isSourceChainSupported = supportedChains.some((chain) => chain.chainId === sourceChainId)
 
 	if (!isSourceChainSupported) {
-		console.log("Unsupported chain - Can't bridge from here")
-		return []
+		throw new Error('Unsupported chain - Can not bridge from here')
```

Comment:
I think this is fine as MVP, but we really should have proper error types, like in elsewhere in the SDK

![image](https://github.com/peanutprotocol/peanut-sdk/assets/12943235/39002b23-a855-4f38-9e35-389cea935b21)

Reply:
my bad, i thought the error refactor hadnt been merged in yet. On it